boostlook.css

```diff
@@ -499,15 +503,14 @@ p, h1, h2, h3, h4, h5, h6 {
 
 .boostlook:not(:has(.doc)) table {
   border-collapse: collapse;
-  margin: 1em;
+  margin: 1em 0;
   border: 1px solid var(--bl-table-border-color);
 }
 
 .boostlook:not(:has(.doc)) th {
   background-color: var(--bl-table-head-background);
   text-align: left;
   padding: 0.25em 0.55em;
-  font-weight: 550;
 }
```

Comment:
Curious why the font-weight was removed.

Reply:
I removed this so the default weight of 700 will display